zstandard/dist/EncoderAndDecoderNodeJS.src.js

```diff
@@ -0,0 +1,134 @@
+﻿(function(global){
```

Comment:
This file is from another distribution I considered using, and should be deleted now that we have compiled our own (much smaller) version.